packtools/sps/formats/oai_dc_agris.py

```diff
@@ -0,0 +1,723 @@
+
+    xml_oai_dc_agris.find('./metadata/{http://purl.org/agmes/1.1/}resources').append(el)
+
+
+def xml_oai_dc_agris_title_pipe(xml_oai_dc_agris, xml_tree):
+    """
+    Schema (https://agris.fao.org/agris_ods/dlio.dtd.txt):
+        <!-- ELEMENT title -->
+        <!ELEMENT dc:title (#PCDATA | dcterms:alternative | ags:titleSupplement)*>
+        <!ATTLIST dc:title
+            xml:lang CDATA #IMPLIED
+        >
+
+    Example:
+        <record>
+            <metadata>
+                <ags:resources>
+                    <ags:resource ags:ARN="XS2021000111">
+                        <dc:title xml:lang="es">
+                            <![CDATA[ La canción reflexiva: en torno al estatuto crítico de la música popular en Brasil ]]>
+                        </dc:title>
+                    </ags:resource>
+                </ags:resources>
+            </metadata>
+        </record>
+    """
+    title = article_titles.ArticleTitles(xml_tree)
+    lang = article_and_subarticles.ArticleAndSubArticles(xml_tree)
+
+    el = ET.Element('{http://purl.org/dc/elements/1.1/}title')
+
+    try:
+        el.set('{http://www.w3.org/XML/1998/namespace}lang', lang.main_lang)
+    except Exception as exc:
+        raise AddLanguageError(f"Unable to add language {exc}")
+
+    try:
+        el.text = title.article_title['text'].strip()
+    except Exception as exc:
+        raise AddTitleError(f"Unable to add title {exc}")
+
+    xml_oai_dc_agris.find(".//ags:resource", {"ags": "http://purl.org/agmes/1.1/"}).append(el)
+
+
+def xml_oai_dc_agris_creator_pipe(xml_oai_dc_agris, xml_tree):
+    """
+    Schema (https://agris.fao.org/agris_ods/dlio.dtd.txt):
+        <!-- ELEMENT creator -->
+        <!ELEMENT dc:creator (#PCDATA | ags:creatorPersonal | ags:creatorCorporate | ags:creatorConference)*>
+        <!ELEMENT ags:creatorPersonal (#PCDATA)>
+        <!ELEMENT ags:creatorCorporate (#PCDATA)>
+        <!ELEMENT ags:creatorConference (#PCDATA)>
+
+    Example:
+        <record>
+            <metadata>
+                <ags:resources>
+                    <ags:resource ags:ARN="XS2021000111">
+                        <dc:creator>
+                            <ags:creatorPersonal>de-Oliveira-Gerolamo, Ismael</ags:creatorPersonal>
+                        </dc:creator>
+                    </ags:resource>
+                </ags:resources>
+            </metadata>
+        </record>
+    """
+    author = article_authors.Authors(xml_tree)
+    try:
+        surname = author.contribs[0].get('surname')
+        given_name = author.contribs[0].get('given_names')
+        author_name = f' {surname.strip()}, {given_name.strip()} '
+        add_creator(xml_oai_dc_agris, author_name.strip())
+    except IndexError:
+        pass
+
+
+def add_publisher(xml_oai_dc_agris, publisher):
+    try:
+        ags = ET.Element('{http://purl.org/agmes/1.1/}publisherName')
+        ags.text = publisher.publishers_names[0].strip()
```

Comment:
@Rossi-Luciano O XML original retorna uma lista de publishers e a DTD do agris aceita mais de 1 publisher, sendo assim, o conteúdo desta função (add_publisher) deveria estar dentro de um `for`.

Reply:
@robertatakenaka, modifiquei o conteúdo da função para considerar mais do que um _publisher_  e, seguindo a recomendação feita no PR #407, removi a função e incorporei seu conteúdo na função `xml_oai_dc_agris_publisher_pipe`.